src/nft/pages/collection/Components.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box } from 'nft/components/Box'
+import { Row } from 'nft/components/Flex'
+import * as styles from 'nft/pages/collection/common.css'
+
+interface ActivitySwitcherProps {
+  showActivity: boolean
+  toggleActivity: () => void
+}
+
+export const ActivitySwitcher = ({ showActivity, toggleActivity }: ActivitySwitcherProps) => {
```

Comment:
This doesn't seem so much like an activity switcher as a general tab manager. Should it be refactored to reflect that?

Reply:
see comments above